tasks/README.rst

```diff
@@ -53,6 +53,8 @@ C) Get creative! Right after the inline comments, add a second door using the el
     else:
         print("You are not so good with numbers, are you?")
 
+See possible solution in :doc:`<adventure_solutions/task_1_if_else.py>`.
```

Comment:
Does it actually work? M When I go to View it doesn't seem so. Maybe this could help
https://github.com/github/markup/issues/259